### PR TITLE
refactor(gh-pages): change the design of the example pages

### PR DESCRIPTION
**Pull request by abarmina**
# PR Details

- change the default container for content pages
- typography and colors updated
- change the design of the example pages

### CODE REVIEW

File: pages/src/common/alert.less [hunk 1]
```diff
@@ -18,14 +20,14 @@
   }
 
   &-warning {
-    color: #856404;
-    background-color: #fff3cd;
-    border-color: #ffeeba;
+    color: @primary-blue;
+    background-color: rgba(0, 151, 231, 0.2);
+    border-color: rgba(0, 151, 231, 0.25)
```
Comment (dshovchko) [inline pages/src/common/alert.less:25]: Look like we need mixin to change color opacity.

File: pages/src/common/flex.less [hunk 2]
```diff
@@ -80,6 +80,18 @@
     max-width: 100%;
   }
 
+  @media @md-xl {
+    &-md-4 {
+      flex: 0 0 33.333333%;
+      max-width: 33.333333%;
```
Comment (dshovchko) [inline pages/src/common/flex.less:86]:
```suggestion
      flex: 0 0 100/3%;
      max-width: 100/3%;
```

**Review by dshovchko — APPROVED** [review]

File: pages/src/common/alert.less (hunk 1 above)
Comment (ala-n) [reply pages/src/common/alert.less:25]: Less have a built-in function to do that, just in case: `fade(@original-color, 50% /*opacity*/)`

**Comment by ala-n** [comment]
Going to merge this branch to the working one (`epic/redesign`) Please still feel free to comment and recommend changes.

**Pull request merged**